itl/command-plugins.conf

```diff
@@ -360,6 +360,10 @@ object CheckCommand "http" {
 			value = "$http_vhost$"
 			description = "Host name argument for servers using host headers (virtual host)"
 		}
+		"--extra-opts" = {
+			value = "$http_extra_opts$"
```

Comment:
This option should probably use `set_if` as well, as there is no need to look for extra options if not explicitly requested by the user.